### PR TITLE
Missing m4

### PR DESCRIPTION
**Pull request by surak**
Same as pull #10695

### CODE REVIEW

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 3 out of 3 (1 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.7.1908, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/d5267b7fd308f3f8d9827e13babf3e5e for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.7.1908, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/f9bee4bfdb673512e26a6a7672fe98d0 for a full test report.

File: easybuild/easyconfigs/n/netCDF-Fortran/netCDF-Fortran-4.5.2-iimpi-2020a.eb [hunk 1]
```diff
@@ -13,6 +13,10 @@ source_urls = ['https://github.com/Unidata/netcdf-fortran/archive/']
 sources = ['v%(version)s.tar.gz']
 checksums = ['0b05c629c70d6d224a3be28699c066bfdfeae477aea211fbf034d973a8309b49']
 
+builddependencies = [
+     ('M4', '1.4.18'),
```
Comment (Micket) [inline easybuild/easyconfigs/n/netCDF-Fortran/netCDF-Fortran-4.5.2-iimpi-2020a.eb:17]:
```suggestion
    ('M4', '1.4.18'),
```

Comment (surak) [reply easybuild/easyconfigs/n/netCDF-Fortran/netCDF-Fortran-4.5.2-iimpi-2020a.eb:17]: On the other file, I could edit from github, but this one I can't

**Pull request closed without merging**